### ISSUE TITLE
Accordion data-allow-multiple & data-allow-toggle not working

### ISSUE THREAD

**Issue by be-codified** [w3c/aria-practices#424]
Hi,

I am trying to use 

```
<dl id="accordionGroup" role="presentation" class="Accordion" data-allow-multiple data-allow-toggle>
```

but nothing happens. Can you please check?

**Comment by nschonni** [comment]
Are you running on an older browser? I found in https://github.com/w3c/aria-practices/pull/422/commits/88c142d48dad51220357104e90c661d14b913889 that the example was using some ES6 stuff

**Comment by be-codified** [comment]
No, I am using latest Chrome or Firefox. @nschonni is script working for you?

**Comment by gerardkcohen** [comment]
@be-codified  @nschonni I have a fix coming soon.

**Comment by mcking65** [comment]
Closed with commit 74887d9.
